### PR TITLE
Remove somalier step from WTS pipeline

### PR DESCRIPTION
**Pull request by skanwal**
Removed somalier from Dragen-WTS pipeline

### CODE REVIEW

File: workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl [hunk 1]
```diff
@@ -338,30 +338,6 @@ steps:
...
-    in:
-      bam_sorted:
-        # The bam from the dragen transcriptome workflow
-        source: run_dragen_transcriptome_step/dragen_bam_out
-      sites:
-        # The VCF output file from the dragen command
-        source: sites_somalier
-      reference:
-        # The reference fasta for the genome somalier
-        source: reference_fasta
```
Comment (alexiswl) [inline workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl:355]: Has 'reference_fasta' been removed from the inputs?

File: workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl [hunk 2]
```diff
@@ -338,30 +338,6 @@ steps:
...
-    label: somalier
-    doc: |
-      Runs the somalier extract function to call the fingerprint on the transcriptome bam file
-    in:
-      bam_sorted:
-        # The bam from the dragen transcriptome workflow
-        source: run_dragen_transcriptome_step/dragen_bam_out
-      sites:
-        # The VCF output file from the dragen command
-        source: sites_somalier
```
Comment (alexiswl) [inline workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl:352]: Has sites_somalier been removed from the inputs?

**Review by alexiswl — CHANGES_REQUESTED** [review]

File: workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl (hunk 1 above)
Comment (skanwal) [reply workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl:355]: We shouldn't remove 'reference_fasta' as it is also required by `arriba_fusion_step`

File: workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl (hunk 2 above)
Comment (skanwal) [reply workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl:352]: updated in https://github.com/umccr/cwl-ica/commit/e746073bd07957de09546a58ed3232f5f1909080

**Review by alexiswl — APPROVED** [review]

**Pull request merged**